CONTRIBUTING.md

```diff
@@ -1,68 +1,85 @@
 # Contributing
 
-## Reporting Issues
+## General guideline
+
+### Reporting issues
```

Comment:
I think I misguided. Let's capitalize section titles: "General Guideline", "Reporting Issues" and so on